python/ray/tests/vsphere/test_vsphere_node_provider.py

```diff
@@ -133,16 +133,6 @@ def test_non_terminated_nodes_with_multiple_filters_not_matching():
     assert len(nodes) == 0
 
 
-def test_is_running():
```

Comment:
This UT is outdated, it passed because “Two negative makes a positive”

I think that function `is_running` is not complicated in logic so should not be mandatory to be covered by UT .